cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TemplateUtils.java

```diff
@@ -31,6 +49,67 @@ public static Map<String, String> labels(HasMetadataTemplate template)   {
         }
     }
 
+    /**
+     * Add additional volumes for a given pod template and a list of existing volumes.
+     *
+     * @param templatePod     The pod template that contains the additional volumes.
+     * @param existingVolumes The list of existing volumes to which the additional volumes will be added.
+     */
+    public static void addAdditionalVolumes(PodTemplate templatePod, List<Volume> existingVolumes) {
+        if (templatePod.getAdditionalVolumes() != null) {
+            templatePod.getAdditionalVolumes().forEach(volumeConfig -> existingVolumes.add(createVolumeFromConfig(volumeConfig)));
+        }
+    }
+
+    /**
+     * Add additional volume mounts to the given list of volume mounts. Validation is performed to ensure none of the
+     * additional volume mount paths are forbidden.
+     * 
+     * @param volumeMounts           The list of volume mounts to be added to
+     * @param additionalVolumeMounts The list of volume mounts to add
+     * @throws RuntimeException If a forbidden mount path is used.
+     */
+    public static void addAdditionalVolumeMounts(List<VolumeMount> volumeMounts, List<VolumeMount> additionalVolumeMounts) {
+        if (additionalVolumeMounts == null) {
+            return;
+        }
+        boolean isSensitivePath = additionalVolumeMounts.stream().anyMatch(additionalVolume -> additionalVolume.getMountPath().startsWith(SENSITIVE_PATH));
```

Comment:
Given that we now check below that the mount path is /mnt, I dont think there is any need for this check here that the mount path is not /tmp

Reply:
I agree and have removed this redundant check.